packages/jam/transition/accumulate/accumulate.ts

```diff
@@ -555,7 +554,10 @@ export class Accumulate {
   private getGasLimit() {
     const calculatedGasLimit =
       GAS_TO_INVOKE_WORK_REPORT * BigInt(this.chainSpec.coresCount) +
-      this.state.privilegedServices.autoAccumulateServices.reduce((acc, { gasLimit }) => acc + gasLimit, 0n);
+      Array.from(this.state.privilegedServices.autoAccumulateServices.values()).reduce(
+        (acc, gasLimit) => acc + gasLimit,
+        0n,
+      );
```

Comment:
```suggestion
const calculatedGasLimit = sumU64(
GAS_TO_INVOKE_WORK_REPORT * BigInt(this.chainSpec.coresCount),
...Array.from(this.state.privilegedServices.autoAccumulateServices.values())
).value;
```
plus a comment why we ignore the overflow? Or additional clamping.